packages/ui/src/TxInfoBar/TxInfoBar.tsx

```diff
@@ -8,25 +8,25 @@ import { ExternalLink } from 'ui/src/Link'
 
 type Props = {
   description: ReactNode
-  txHash: string | string[]
+  txHash: string | string[] | undefined
   onClose?: () => void
 }
 
 const TxInfoBar = ({ description, txHash, onClose }: Props) => (
   <StyledInfoBar grid gridTemplateColumns="1fr auto" gridColumnGap="3" flexAlignItems="center" fillWidth>
     <InfoTitle>
       {description}{' '}
-      {Array.isArray(txHash) ? (
-        txHash.map((tx) => (
-          <StyledExternalLink href={tx} key={tx}>
-            <RCExternal />
-          </StyledExternalLink>
-        ))
-      ) : (
-        <StyledExternalLink href={txHash}>
-          <RCExternal />
-        </StyledExternalLink>
-      )}
+      {Array.isArray(txHash)
+        ? txHash.map((tx) => (
```

Comment:
I know you don't like lodash but we can easily implement [this](https://lodash.com/docs/4.17.15#castArray) ourselves too
```suggestion
        castArray(txHash).map((tx) => (
```